### DISCUSSION TITLE
option for vertical text, right to left.

### DISCUSSION THREAD

**Discussion by MooMeowCat** [sumatrapdfreader/sumatrapdf#3274] [Ideas]
Just a preference I have for Chinese novels.  I tried turning on manga mode which reversed the page flipping, but the text still goes horizontal and left to right.

**Comment by mark10als** [comment]
I think your problem has the same underlying cause as mine (#3277).
https://github.com/sumatrapdfreader/sumatrapdf/discussions/3277
The SumatraPDF project is changing the core module to MuPDF.
Few core modules (core engines) support vertical writing and right binding.
The PDF is displayed correctly because it specifies the position of each character or uses the one converted to an image.
Manga (CBZ) has "manga mode" in the menu, but PDF does not.
* "Manga mode" is a display method for right-bound books.
I think that Electron, the core module (core engine), is the best for vertical writing and right binding.
